target/linux/ar71xx/base-files/etc/board.d/01_leds

```diff
@@ -182,7 +182,8 @@ cf-e320n-v2)
 	ucidef_set_led_wlan "wlan" "WLAN" "$board:blue:wlan" "phy0tpt"
 	;;
 cf-e380ac-v1|\
-cf-e380ac-v2)
+cf-e380ac-v2)\
+cf-e355ac)
 	ucidef_set_led_netdev "lan" "LAN" "$board:green:lan" "eth0"
 	ucidef_set_led_wlan "wlan2g" "WLAN2G" "$board:blue:wlan2g" "phy1tpt"
```

Comment:
> Yes, I'm testing the code, but with the LEDs there is a problem. It is a single RGB led that changes color, so it is difficult to see if it works well.

You can manipulate with LEDs manually, for example `echo 0 > /sys/class/leds/...led-name-here.../brightness`. With this you can manually shut down two LEDs and check if 3rd one works correctly (trigger, active low/high, etc.), then reload LEDs setting with `/etc/init.d/led reload` and repeat for other two LEDs.